docker/entrypoint.sh

```diff
@@ -86,6 +86,7 @@ case "$1" in
             # Starting at a cached Zebra tip, test a JSON-RPC call to Zebra.
             ls -lh "$ZEBRA_CACHED_STATE_DIR"/*/* || (echo "No $ZEBRA_CACHED_STATE_DIR/*/*"; ls -lhR  "$ZEBRA_CACHED_STATE_DIR" | head -50 || echo "No $ZEBRA_CACHED_STATE_DIR directory")
             cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --package zebrad --test acceptance -- --nocapture --include-ignored fully_synced_rpc_test
+            cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --package zebrad --test acceptance -- --nocapture --include-ignored rpc_z_getsubtreesbyindex_sync_snapshots
```

Comment:
I think the test runner is stopping after the first test, due to this workflow code:
https://github.com/ZcashFoundation/zebra/blob/a7501c16b1c9a3531f3cf833b10d1fcdb98f6393/.github/workflows/deploy-gcp-tests.yml#L597-L599

This might also be the cause of bug #7564.

So it might be easier to run both tests at the same time. That will need a change to the test name, I'll make another suggestion below.

```suggestion
            # Run both the fully synced RPC test and the subtree snapshot test
            cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --package zebrad --test acceptance -- --nocapture --include-ignored fully_synced_rpc_
```

Reply:
Thanks. https://github.com/ZcashFoundation/zebra/pull/7515/commits/2ebb28032288e41ea80176b1ddcee59208cfcb28